tests/testthat/_snaps/compat-types-check.md

```diff
@@ -292,7 +310,7 @@
     Output
       <error/rlang_error>
       Error in `checker()`:
-      ! `foo` must be a symbol, not a string.
+      ! `foo` must be a symbol, not "foo".
```

Comment:
If we're now inlining strings, do we need to worry about very long strings?

Reply:
hmm good point. We should also worry about strings with newlines.

Reply:
Maybe truncate at 30 chars and apply `encodeString()`?